identity_fips_openssl/attributes/default.rb

```diff
@@ -4,6 +4,6 @@
 default['identity_fips_openssl']['fips']['checksum']           = 'a3cd13d0521d22dd939063d3b4a0d4ce24494374b91408a05bdaca8b681c63d4'
 default['identity_fips_openssl']['openssl']['version']         = default.fetch(:identity_shared_attributes).fetch(:openssl_version)
 default['identity_fips_openssl']['openssl']['prefix']          = "/opt/openssl-#{default['identity_fips_openssl']['openssl']['version']}"
-default['identity_fips_openssl']['openssl']['url']             = "https://www.openssl.org/source/openssl-#{default['identity_fips_openssl']['openssl']['version']}.tar.gz"
-default['identity_fips_openssl']['openssl']['checksum']        = '14cb464efe7ac6b54799b34456bd69558a749a4931ecfd9cf9f71d7881cac7bc'
+default['identity_fips_openssl']['openssl']['url']             = "https://www.openssl.org/source/old/1.0.2/openssl-#{default['identity_fips_openssl']['openssl']['version']}.tar.gz"
```

Comment:
This URL change shouldn't be necessary -- I just tested it without the `old/1.0.2/` path and the file is there, along with the same checksum.

Reply:
There were a couple of days where the previous url was unavailable and could not build.  The 1.0 releases are no longer supported and it appears this will be the final resting place.  There is no chance we will start using the 1.1 release as it will not have a FIPS module.